src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -208,19 +208,9 @@ public string Name
 
         /// <summary>
         /// The original name of the node.
-        /// This is the name of the node as it is when first added to the canvas.
         /// </summary>
         [JsonIgnore]
-        public string OriginalName
-        {
-            get 
-            {
-                if (nodeLogic.IsCustomFunction)
-                    return nodeLogic.GetCustomNodeOriginalName();
-
-                return nodeLogic.GetOriginalName(); 
-            }
-        }
+        public string OriginalName => nodeLogic.GetOriginalName();
```

Comment:
it's not clear to me if our build machine support expression body members yet, best to stick with the older syntax for now.